addons/buildables/functions/fnc_areaInit.sqf

```diff
@@ -0,0 +1,66 @@
+#include "script_component.hpp"
+/*
+ * Author: veteran29
+ * Inits buildable area.
+ *
+ * Arguments:
+ * 0: Editor layer name <STRING>
+ * 1: Buildable area <STRING, ARRAY>
+ * 2: Area display name <STRING>
+ * 3: Amount of required supplies <NUMBER, defaults to 1>
+ * 4: How long it takes to build the area <NUMBER, defaults to 60>
+ * 5: Should layer objects be hidden <BOOL, defaults to false>
+ * 6: Class names of objects that can be used to supply the area <BOOL, defaults to false>
+ *
+ * Return Value:
+ * Area was initialized <BOOL>
+ *
+ * Example:
+ * ["checkpoint_1", "checkpoint_1_area"] call afmf_buildables_fnc_areaInit
+ *
+ * Public: Yes
+ */
+
+if (!isServer) exitWith {false};
+
+params [
+    ["_layer", "", [""]],
+    ["_area", "", ["", []]],
```

Comment:
This param name is a bit confusing as in other functions it is name for hash map, and here this is string/array. Also, what does this parameter do? Is it a name of the module?

Reply:
It's the "area"/coordinates/marker where you can deliver the supplies. I can rename this to "zone"

Reply:
I think it can also be a `_deliveryArea` or something. It's up to you.